basic2/upload.php

```diff
@@ -0,0 +1,33 @@
+<?php
+$errorMessage = "";
+$displayName = "";
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+    if (isset($_FILES["image"])) {
+        $targetDir = "uploads/";
+        $targetFile = $targetDir . basename($_FILES["image"]["name"]);
+        $uploadOk = 1;
+        $imageFileType = strtolower(pathinfo($targetFile, PATHINFO_EXTENSION));
+
+        //Check image using getimagesize function and get size. If a valid number is obtained then uploaded file is an image.
```

Comment:
A comment should contain only 80 characters in a single line, if exceeds then write it in the next line.